Source/common/SNTConfigurator.h

```diff
@@ -132,12 +132,6 @@ extern NSString *const kDefaultConfigFilePath;
 @property(readonly, nonatomic) NSURL *syncBaseURL;
 
 ///
-///  If YES, mid-execution event uploads are skipped.
-///  This property is never stored on disk.
-///
-@property BOOL syncBackOff;
```

Comment:
It's not being used currently. I do need to add a similar feature to the sync manager. Will do so in a separate pull.